frontends/ol-learning-resources/src/components/LearningResourceCard.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { useCallback } from "react"
```

Comment:
Do we need a version of LearningResourceCard both here and in infinite-corridor/components? Also is this LearningResourceCard used anywhere? the Home.tsx uses the other one